38.JS_Async/assets/js/main.js

```diff
@@ -0,0 +1,201 @@
+const API_KEY = "7a4aabbe";
+const API_URL = `http://www.omdbapi.com/?apikey=${API_KEY}`;
+
+const fetchData = async (query, type, page = 1) => {
+  return await fetch(`${API_URL}&s=${query}&type=${type}&page=${page}`).then(
```

Comment:
try to use axios and using then this await is a bed practise
use two times await instead